corehq/apps/app_manager/static/app_manager/js/details/screen_config.js

```diff
@@ -219,11 +219,45 @@ hqDefine('app_manager/js/details/screen_config', function () {
             is_parent: true,
         });
         self.moduleId = ko.observable(init.moduleId || (defaultModule ? defaultModule.unique_id : null));
-        self.active = ko.observable(init.active);
+        self.allCaseModules = ko.observable(init.allCaseModules);
         self.parentModules = ko.observable(init.parentModules);
         self.lang = ko.observable(init.lang);
         self.langs = ko.observable(init.langs);
 
+        self.selectOptions = [
+            {id: 'none', text: gettext('None')},
+            {id: 'parent', text: gettext('Parent')},
+            {id: 'other', text: gettext('Other')},
+        ];
+        if (init.active){
+            if (init.relationship == 'parent') {
+                var selectedMode = 'parent';
+            }
+            else {
+                var selectedMode = 'other';
+            }
+        }
+        else {
+            var selectedMode = 'none';
+        }
+        self.selectMode = ko.observable(selectedMode);
```

Comment:
Nitpick: maybe name these the same? The difference is so slight it's potentially confusing.

Reply:
Sure